test/aspnet-core-demo/AbpAspNetCoreDemo/Views/AlertsTest/Index.cshtml

```diff
@@ -26,3 +30,22 @@
         }
     </div>
 }
+
+@section scripts{
+    <script src="~/lib/toastr/toastr.min.js" type="text/javascript"></script>
+    <script src="~/Abp/Framework/scripts/libs/abp.toastr.js" type="text/javascript"></script>
+
+    @if (AlertManager.Alerts.Any(a => a.DisplayType == AlertDisplayType.Toastr))
+    {
+        <script type="text/javascript">
+        @foreach (var toastrMessage in AlertManager.Alerts.Where(a => a.DisplayType == AlertDisplayType.Toastr))
+        {
+            @Html.Raw(string.Format("abp.notify.{0}('{1}','{2}', {{ 'closeButton':  {3} }});",
+               toastrMessage.Type.ToString().ToLowerInvariant().Replace("warning", "warn").Replace("danger", "error"),
```

Comment:
its not seems good beacuse of alertType names not matching with notify names.
Any other idea?